metadata-ingestion/src/datahub/ingestion/source/hex/api.py

```diff
@@ -87,8 +87,9 @@ class HexApiAccessType(StrEnum):
     """Access type enum."""
```

Comment:
are we using the access type metadata anywhere? 

BaseModel will ignore unknown fields. So if we don't need it, we can just not map it at all

Reply:
indeed
addressed in commit 0fd7dc1